torch/nn/modules/conv.py

```diff
@@ -10,7 +11,8 @@
 class _ConvNd(Module):
 
     def __init__(self, in_channels, out_channels, kernel_size, stride,
-                 padding, dilation, transposed, output_padding, groups, bias):
+                 padding, dilation, transposed, output_padding, groups, bias,
+                 initializer):
```

Comment:
Sorry, this default can be passed in, in the actual implementations i.e. `Conv1d` etc